content/post/2020/01/mannequiny-released/index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Mannequiny: the 3D character controller is released"
+description = "Open source 3D character controller for the Godot engine"
+author = "razoric"
+
+date = 2020-01-17T15:00:00+00:00
+weight = 5
+
+categories = ["news"]
+tags = ["gdquest", "godot", "godotengine", "asset", "3D", "controller", "character"]
+
+[[resources]]
+name = "banner"
+src = "banner.png"
++++
+
+As part of the [2019 Godot Kickstarter campaign](https://www.kickstarter.com/projects/gdquest/create-your-own-games-with-godot-the-free-game-eng), one of the stretch goals reached was a 3D character controller kit to help quickly prototype 3D games with an animated character. This project became Mannequiny, the Open 3D Mannequin for Godot.
+
+The mannequin was made by [Luciano Muñoz](https://twitter.com/lucianomunoz_). The game code was built by Josh aka [Cheeseness](https://twitter.com/ValiantCheese) and Francois Belair aka [Razoric](https://twitter.com/Razoric480). It is fully open sourced under the MIT license and open to contributions at its [repository](https://github.com/GDquest/godot-3d-mannequin/). The 3D character is professionally animated, the controller works with both mouse+keyboard and gamepad, and it is supported by a camera rig that supports zoom-to-aim, auto-rotation, and level geometry collision zooming.
+
+Version 0.2.0 is available to download as a [repository release](https://github.com/GDQuest/godot-3d-mannequin/releases/tag/v0.2.0) (including the Blender file), or on the [Godot asset library](https://godotengine.org/asset-library/asset/440).
+
+Additionally, you can learn how the character was built using our [3D character controller course](https://www.gdquest.com/post/2019/12/godot-3d-course-ea/)! Follow along to see how every piece came together.
+
+Thank you to all our backers who supported this open source effort, and we hope to see this be useful to many among you!
+
+See you next time, and keep working!
+
+![Screenshot of Mannequiny][image-mannequiny]

```

Comment:
you can link straight to images with parentheses: `![Screenshot of Mannequiny](./mannequiny.png)`